src/tests/utils/database-helpers.ts

```diff
@@ -102,7 +112,8 @@ export default class DatabaseHelpers {
     const content = note.content ?? '{}';
     const publicId = note.publicId ?? createPublicId();
 
-    // eslint-disable-next-line
+    /** eslint disabled because metadata variable is never user */
+    /* eslint-disable-next-line */
```

Comment:
its a bad practice to disable all eslint rules when you need to mute just one. Specify a rule to disable explicitly.

Bytheway, I think you won't need to disable that rule if you will use the `_` name for unused variable

Reply:
![image](https://github.com/codex-team/notes.api/assets/130844513/752cf919-607b-4a96-961e-2dccb9f0e913)
it treats underline like variable, will specify a rule

Reply:
Seems like we need to configure it in our [esling config](https://github.com/codex-team/eslint-config/blob/master/ts.json#L20): 

https://stackoverflow.com/questions/64052318/how-to-disable-warn-about-some-unused-params-but-keep-typescript-eslint-no-un